### DISCUSSION TITLE
The red dot does not disappear when the mouse quickly slides from the editor page to the left page

### DISCUSSION THREAD

**Discussion by xiaguangbo** [microsoft/vscode-cpptools#9492] [General]
vscode itself has no debugging function for .c files, and there is no red dot on the left side of the editor's line number

After installing the C/C++ extension, under normal circumstances, when the mouse moves to the left of the line number, a dark red dot will appear. When the mouse continues to move left beyond the editing page, the dark red dot will disappear, but when you quickly swipe across, there is a chance that the red dot will not disappear

<img width="850" alt="image" src="https://user-images.githubusercontent.com/102402151/175035164-96205dc7-204a-49e4-92ff-14be15f32895.png">

**Comment by sean-mcmanus** [comment]
It sounds like you're describing a UI bug with VS Code. Our extension doesn't implement any of the UI layer which could cause UI bugs. You should file a bug at https://github.com/microsoft/vscode/issues .

**Reply by xiaguangbo** [reply]
ok，thanks
